### PR TITLE
Fix unset environment variable in harbor-trivy image

### PR DESCRIPTION
**Pull request by vincentni**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/goharbor/harbor/docker/linux/harbor-trivy/Dockerfile [hunk 1]
```diff
@@ -22,7 +21,7 @@ RUN yum install -y shadow-utils >> /dev/null \
 
 HEALTHCHECK --interval=30s --timeout=10s --retries=3 CMD curl --fail -s http://localhost:8080/probe/healthy || curl -k --fail -s https://localhost:8443/probe/healthy || exit 1
 
-ENV TRIVY_VERSION=${TRIVY_VERSION}
+ENV TRIVY_VERSION=v0.28.1
```
Comment (jaxesn) [inline projects/goharbor/harbor/docker/linux/harbor-trivy/Dockerfile:24]: Why not pass this in?

File: projects/goharbor/harbor/Makefile [hunk 2]
```diff
@@ -47,8 +47,8 @@ SOURCE_PATTERNS=./core ./jobservice ./registryctl ./cmd/migrate-patch ./cmd/stan
 BINARY_TARGET_FILES_BUILD_ALONE=harbor-core
 GO_MOD_PATHS=src src src src src src
 
-HARBOR_BASE_NAMESPACE=goharbor
-IMAGE_BUILD_ARGS=HARBOR_BASE_NAMESPACE
```
Comment (jaxesn) [inline projects/goharbor/harbor/Makefile:51]: Is this one not needed?

Comment (vincentni) [reply projects/goharbor/harbor/Makefile:51]: no, it is not needed.

**Review by jaxesn — APPROVED** [review]

**Pull request merged**